doc/install.txt

```diff
@@ -412,7 +412,7 @@ that we use.
 3) Install the ATLAS library. ATLAS is an open source optimized version of
 BLAS. You can install a precompiled version on most OSes, but if you're willing
 to invest the time, you can compile it to have a faster version (we have seen
-speed-ups of up to 3x, especialy on more recent computers, against the
+speed-ups of up to 3×, especially on more recent computers, against the
```

Comment:
> Martin, can you do that small change?

I've changed the times symbol back to x and pushed it. Should I do anything else?
(I only have very little experience with GitHub / git).